app.js

```diff
@@ -1,15 +1,25 @@
+const persist = require('choo-persist')
 const mount = require('choo/mount')
 const log = require('choo-log')
+const xtend = require('xtend')
 const choo = require('choo')
 
-const mainView = require('./pages/main')
+const opts = {
+  filter: (state) => {
+    state = xtend(state)
```

Comment:
I think while your suggestion might be easier, it's not simpler - I don't like `Object.assign()` for the reason that it doesn't work everywhere and polyfills are large; I'd rather leave `extend` in there since we're already using it in other places too. I'd suggest we moderate ourselves with the usage of ES6 given most of the team barely uses any of it also